client/src/app/site/pages/meetings/pages/motions/pages/motion-polls/components/motion-poll-detail/motion-poll-detail.component.ts

```diff
@@ -35,7 +35,13 @@ export class MotionPollDetailComponent extends BasePollDetailComponent<ViewMotio
     }
 
     protected createVotesData(): BaseVoteData[] {
```

Comment:
Instead of extending `BaseVoteData` (which is also used in other places) add a new type specific for this use case.

```suggestion
    protected createVotesData(): BaseVoteData[] {
```